src/checkActivities.rb

```diff
@@ -0,0 +1,143 @@
+require 'colorize'
+require 'git'
+
+module ActivityCheck
+
+    def self.initialize(activityDirName)
+        @activityDirName = activityDirName
+
+        # create js file name array
+        @jsFilesArray = Dir.glob("../src/activities/" + @activityDirName + "/*.js")

```

Comment:
it should be recursive inside the folder (for example, we can have for some activities, @activityDirName + "/resource/boards/*qml" files